### DISCUSSION TITLE
Use lldap for server login?

### DISCUSSION THREAD

**Discussion by benisai** [lldap/lldap#967] [Q&A]
Hi guys, is it possible to use LLDAP as a server login? I didn't see a example config, so I figured I'd ask people much smarter than me. 

I have a few Ubuntu servers and would like 1 common username/password. 

Any help would be appreciated 👍☺️

**Comment by nitnelave** [comment]
You might want to look at the PAM guide: https://github.com/lldap/lldap/blob/main/example_configs%2Fpam%2FREADME.md
